tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -371,6 +378,13 @@ Then('file/folder {string} should not be listed on the webUI', async function (f
   )
 })
 
+Then('{string} {string} should not be listed on the webUI', async (elementType, folder) => {
```

Comment:
I think that will be too confusing to have a very similar step one with quotes around "folder" and one without.
Better to have separate step for file and folder